completion/bash/pkgdev

```diff
@@ -0,0 +1,68 @@
+# bash completion for pkgdev
+
+_pkgdev() {
+    local i=1 cmd cur prev words cword split
+    _init_completion || return
+
+    if [[ ${prev} = "--color" ]]; then
+        COMPREPLY=($(compgen -W "true false" -- "$cur"))
+        return
+    fi
+    COMPREPLY=($(compgen -W "--help --debug --quiet --verbose --color" -- "${cur}"))
+
+    # find the subcommand
+    while [[ "${i}" -lt "${COMP_CWORD}" ]]; do
+        local s="${COMP_WORDS[i]}"
+        case "${s}" in
+            -*) ;;
+            *)
+                cmd="${s}"
+                break
+                ;;
+        esac
+        ((i++))
+    done
+
+    if [[ "${i}" -eq "${COMP_CWORD}" ]]; then
+        COMPREPLY+=($(compgen -W "commit manifest mask push --version" -- "${cur}"))
+        return
+    fi
+
+    case "${cmd}" in
+        commit)
+            case "${prev}" in
+                -[bcTm] | --bug | --closes | --tag | --message)
+                    COMPREPLY=()
+                    ;;
+                -M | --message-template)
+                    COMPREPLY=($(compgen -f -- "${cur}"))
+                    ;;
+                -s | --scan | --mangle)
+                    COMPREPLY=($(compgen -W "true false" -- "${cur}"))
+                    ;;
+                *)
+                    COMPREPLY+=($(compgen -W "--bug --closes --tag --dry-run --scan --ask --mangle --message --message-template --update --all" -- "${cur}"))
```

Comment:
As an example of what I mean, instead of verbosely listing all option for ``pkgdev commt`` here, instead store them in a more maintainable array and then use ``"${COMMIT_OPTIONS[@]}"`` or similar here. Each option (and potential short option if it exists) should be on its own line for each option group array.